billit/client.py

```diff
@@ -46,9 +46,12 @@ def _handle_response(self, response):
             error = f"{response.text}"
             if "application/json" in response.headers["Content-Type"]:
                 resp = response.json()
-                error = f"Message: {resp['message']}, Error details: {resp.get('errors', None)}, {resp.get('data', None)}"
+                if "message" in resp:
+                    error = f"Message: {resp.get('message')} , Error details: {resp.get('errors')}, {resp.get('data')}"
+                elif "msg" in resp:
+                    error = f"Message: {resp.get('msg')} , Error details: {resp.get('errors')}, {resp.get('data')}"
```

Comment:
We could use early raises here, to make code reading a bit easier.

```suggestion
                if "message" in resp:
                    error = f"Message: {resp.get('message')} , Error details: {resp.get('errors')}, {resp.get('data')}"
                    raise APIError(error, response)

                if "msg" in resp:
                    error = f"Message: {resp.get('msg')} , Error details: {resp.get('errors')}, {resp.get('data')}"
                    raise APIError(error, response)
```